lib/auth/clt.go

```diff
@@ -759,46 +715,100 @@ func (c *Client) ValidateOIDCAuthCallback(q url.Values) (*OIDCAuthResponse, erro
 	return response, nil
 }
 
-type chunkRW struct {
-	c  *Client
-	id string
+// EmitAuditEvent sends an auditable event to the auth server (part of evets.AuditLogI interface)
+func (c *Client) EmitAuditEvent(eventType string, fields events.EventFields) error {
+	_, err := c.PostJSON(c.Endpoint("events"), &auditEventReq{
+		Type:   eventType,
+		Fields: fields,
+	})
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	return nil
 }
 
-func (c *chunkRW) ReadChunks(start int, end int) ([]recorder.Chunk, error) {
-	out, err := c.c.Get(c.c.Endpoint("records", c.id, "chunks"), url.Values{
-		"start": []string{strconv.Itoa(start)},
-		"end":   []string{strconv.Itoa(end)},
-	})
+// PostSessionChunk allows clients to submit session stream chunks to the audit log
+// (part of evets.AuditLogI interface)
+//
+// The data is POSTed to HTTP server as a simple binary body (no encodings of any
+// kind are needed)
+func (c *Client) PostSessionChunk(sid session.ID, reader io.Reader) error {
```

Comment:
may be `WriteSessionChunk` or `POSTSessionChunk` as `Post` does not conform to code review guidelines for initialisms